src/containers/AccountConnection.jsx

```diff
@@ -266,7 +285,7 @@ class AccountConnection extends Component {
                   />
                 )}
               </ul>}
-            {!connector.dataType &&
+            {!(connector.dataType && connector.dataType.length) &&
```

Comment:
Yep, I will do this last one (agree about not reused function), thanks @enguerran and @gregorylegarec 👍